stable/fluent-bit/templates/daemonset.yaml

```diff
@@ -152,3 +157,8 @@ spec:
         hostPath:
           path: /mnt
 {{ end }}
+{{ if .Values.on_microk8s }}
+      - name: varsnapmicrok8scommonvarlibdockercontainers
+        hostPath:
+          path: /var/snap/microk8s/common/var/lib/docker/containers
+{{ end }}
```

Comment:
Same here. do you need /var/lib/containers? That can be symlinks I think so you might need both.